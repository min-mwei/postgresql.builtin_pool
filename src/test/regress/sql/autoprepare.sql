select count(*) from pg_class where relname='pg_class';
select * from pg_autoprepared_statements;
set autoprepare_threshold = 1;
select count(*) from pg_class where relname='pg_class';
select * from pg_autoprepared_statements;
select count(*) from pg_class where relname='pg_class';
select * from pg_autoprepared_statements;
set autoprepare_threshold = 0;
select * from pg_autoprepared_statements;
select count(*) from pg_class where relname='pg_class';
select * from pg_autoprepared_statements;
