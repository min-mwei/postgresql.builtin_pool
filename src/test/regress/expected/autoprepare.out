select count(*) from pg_class where relname='pg_class';
 count 
-------
     1
(1 row)

select * from pg_autoprepared_statements;
 statement | parameter_types | exec_count 
-----------+-----------------+------------
(0 rows)

set autoprepare_threshold = 1;
select count(*) from pg_class where relname='pg_class';
 count 
-------
     1
(1 row)

select * from pg_autoprepared_statements;
                        statement                        | parameter_types | exec_count 
---------------------------------------------------------+-----------------+------------
 select count(*) from pg_class where relname='pg_class'; | {unknown}       |          1
 select * from pg_autoprepared_statements;               | {}              |          1
(2 rows)

select count(*) from pg_class where relname='pg_class';
 count 
-------
     1
(1 row)

select * from pg_autoprepared_statements;
                        statement                        | parameter_types | exec_count 
---------------------------------------------------------+-----------------+------------
 select count(*) from pg_class where relname='pg_class'; | {unknown}       |          2
 select * from pg_autoprepared_statements;               | {}              |          2
(2 rows)

set autoprepare_threshold = 0;
select * from pg_autoprepared_statements;
 statement | parameter_types | exec_count 
-----------+-----------------+------------
(0 rows)

select count(*) from pg_class where relname='pg_class';
 count 
-------
     1
(1 row)

select * from pg_autoprepared_statements;
 statement | parameter_types | exec_count 
-----------+-----------------+------------
(0 rows)

